### PR TITLE
Add Bluesky, Github & Strava - tweaks Youtube, Tiktok and Twitch

### PR DESCRIPTION
**Pull request by bluefroguk1**
Add Bluesky and Github Options

### CODE REVIEW

**Comment by bluefroguk1** [comment]
Updated to title up title,
add @ on tiktok and youtube usernames for the urls to work
set twitch to work by username rather than url

**Comment by bluefroguk1** [comment]
added Strava

File: about.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "discourse-social-links-clickable",
+  "name": "Social Icons on User Profile",
```
Comment (weallwegot) [inline about.json:2]: i can't remember a ton of the details
here but does this change discoverability at all

File: javascripts/discourse/api-initializers/social-links-clickable.js [hunk 2]
```diff
@@ -2,11 +2,15 @@ import { apiInitializer } from "discourse/lib/api";
 import SocialLinksClickable from "../components/social-links-clickable";
 
 export default apiInitializer("1.13.0", (api) => {
+  // Render the SocialLinksClickable component in the specified outlets
```
Comment (weallwegot) [inline javascripts/discourse/api-initializers/social-links-clickable.js:5]: thank you for adding these 🙌🏿

File: javascripts/discourse/components/social-links-clickable.gjs [hunk 3]
```diff
@@ -1,18 +1,22 @@
 import Component from "@glimmer/component";
 import { inject as service } from "@ember/service";
-import icon from "discourse-common/helpers/d-icon";
+import icon from "discourse-common/helpers/d-icon"; // Assuming this helper is valid.
 
-export default class SocialLinksClikable extends Component {
+export default class SocialLinksClickable extends Component {
```
Comment (weallwegot) [inline javascripts/discourse/components/social-links-clickable.gjs:5]: 😅 lol thanks for fixing typo

**Review by weallwegot — APPROVED** [review]

File: about.json (hunk 1 above)
Comment (bluefroguk1) [reply about.json:2]: Doesn't effect discoverability (in meta etc.) - but makes it tidier in the user's Theme Component screen

File: javascripts/discourse/components/social-links-clickable.gjs (hunk 3 above)
Comment (bluefroguk1) [reply javascripts/discourse/components/social-links-clickable.gjs:5]: thank my IDE for spotting it tbh

**Pull request merged**